pages/implementation/property-groups-unfolding.mdx

```diff
@@ -0,0 +1,40 @@
+---
+layout: 'guide'
+title: 'Start using Avo Visual Debugger in code'
+---
+
+import Link from '../../components/Link';
+
+In Avo you can group multiple event properties into a property group.
+
+Then you can add the whole group to selected events.
+
+For example in Avo we have a property group called "Source", that includes 2 properties: "Source Id" and "Source Name".
+
+We add this property group to any event that is related to a source instead of adding those 2 properties manually every time.
+
+Property groups can include more than 2 properties, there is no upper limit.
+
+Read more about property groups <Link passHref href="/workspace/tracking-plan/properties#a-nameproperty-groupsa-event-property-groups"><a>here</a></Link>.
+
+## Property groups in the Codegen
+
+When generating code for a tracking plan with property groups the property groups can be in 2 states:
```

Comment:
Could you add an explanation of when they're Folded and when they're Unfolded?

Reply:
Added